### PR TITLE
[tests] rename no_ft_funcs to NO_FT_FUNCS

### PR DESCRIPTION
**Pull request by ebraminio**
As https://github.com/harfbuzz/harfbuzz/commit/43e3ab019c6b6ab8e429b3dd3aa13a7b9e8316e6#r40363339

@khaledhosny would you please have a look? Thanks :)

### CODE REVIEW

File: test/shaping/data/aots/Makefile.am [hunk 1]
```diff
@@ -15,7 +15,12 @@ EXTRA_DIST = \
 	$(NULL)
 
 TEST_EXTENSIONS = .tests
-TESTS_LOG_COMPILER = $(srcdir)/../../run-tests.py $(top_builddir)/util/hb-shape$(EXEEXT)
+
+if HAVE_FREETYPE
+TESTS_LOG_COMPILER = HAVE_FREETYPE=1 $(srcdir)/../../run-tests.py $(top_builddir)/util/hb-shape$(EXEEXT)
```
Comment (ebraminio) [inline test/shaping/data/aots/Makefile.am:20]: oh, this won't work, hmmm, guess some help would be nice.

**Comment by ebraminio** [comment]
dropped HAVE_FREETYPE https://github.com/harfbuzz/harfbuzz/commit/fbad4d8ca104395d3cbf136dd3387f708b1630fc and just renamed the environment variable https://github.com/harfbuzz/harfbuzz/pull/2535/commits/ca74dc3630e351b4d16f710d1c9fc6e9b9e62206 just to address Khaled's comment.

**Comment by ebraminio** [comment]
Ok merged as it now is very simple change, please let me know any other thing with the changes and will try my best to improve them. Thank you so much :-)

**Comment by behdad** [comment]
I'm confused. Why not pass `HAVE_FREETYPE` as you originally did? Guess because you wanted running run-tests manually to assume FreeType available. We can default to 1 if envvar not set. Anyway, either way works. We'll clean up later after removing autotools.  Thanks.

**Comment by ebraminio** [comment]
> Why not pass `HAVE_FREETYPE` as you originally did?

To be honest I just didn't know how to pass environment variable to `TESTS_LOG_COMPILER`, as it was complaining about `../../../../test-driver: line 107: HAVE_FREETYPE=1: not found`, I thought `TESTS_LOG_COMPILER` is just a simple command.

> Anyway, either way works. We'll clean up later after removing autotools.

Exactly.

**Comment by behdad** [comment]
See `TESTS_ENVIRONMENT` in api tests.

**Comment by ebraminio** [comment]
> See `TESTS_ENVIRONMENT` in api tests.

Ah, let's go for it then.

**Pull request merged**